webapp/backend/src/server.ts

```diff
@@ -0,0 +1,23 @@
+// Import libraries
+import express from "express";
```

Comment:
can we make use of destructuring here and import exactly what's needed from the express library so we don't import all of it? eg
```
import {Application, Request, Response} from "express"
```

that way we reduce the import size and also make the typescript types straightforward. eg
```
(req: Request, res: Response)
```

more details https://dev.to/dianjuar/importing-modules-in-javascript-are-we-doing-it-right-nc

Reply:
Great. Well noted. I have made the changes. However, with the import for express, express() works only when express is added.